lib/open-amp/resource_table.h

```diff
@@ -16,13 +16,20 @@ extern "C" {
 
 #if (CONFIG_OPENAMP_RSC_TABLE_NUM_RPMSG_BUFF > 0)
 
-#define VDEV_ID                 0xFF
+#define VDEV_ID 0xFF
+#ifdef CONFIG_IPM_RX_CHANNEL_ID
+#define VRING0_ID CONFIG_IPM_RX_CHANNEL_ID
+#else
 #define VRING0_ID 0 /* (master to remote) fixed to 0 for Linux compatibility */
+#endif
+#ifdef CONFIG_IPM_TX_CHANNEL_ID
```

Comment:
Add a new line here for visibility to separate TX and RX definitions.

Reply:
done